css/main.css

```diff
@@ -325,3 +325,49 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact_form_section {
+  height: 100vh;
+  display: flex;
+}
+
+.contact_form_header {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  max-width: 20vw;
+  margin: 50px 30px 0 10vw;
+}
+
+.form_container {
+  margin: 50px 0 0 0;
+  width: 40vw;
+}
```

Comment:
- [ ] **[REQUIRED CHANGES]** As stated in the last review and in the project requirements, it is important that you implement the `display:flex` to your sections, to allow you to improve in flex styling. The recommended tutorial mentioned in the last review is a great starting point if you're having difficulties with it.

- [ ] **[OPTIONAL]** I would also like to mention that **_if_** you feel like you have some extra time, try to work on the optional changes as suggested. They will help you to better understand the flexbox behavior, and will also make your project look more professional for whoever checks it!